### PR TITLE
Complete manage-admissions endpoint

### PR DESCRIPTION
**Pull request by norbye**
PR to enable the creation and editing of admissions from the website (should remove the need for shelling<3).
## Tasks
- [x] Create django endpoints
- [x] Connect the frontend to the endpoints
- [x] Connect groups to the endpoint
- [x] Add visual feedback to the frontend

## Noteworthy stuffs
* Webkom + the leaders of HS, revue and backup have access to create admissions
* Webkom can edit any admission, the others can edit admissions they themselves have created
* Removed Django admin from /api/admin route
* Setup custom routes in /api/admin to start separating all endpoints (with the purpose of allowing for more intuitive endpoint-names, further work will be done to solve ABA-218)

Smaller less noteworthy stuff
* The error messages in the frontend are shit, but to update it the api implementation in the frontend has to be improved and I don't want to do it in this PR
* Removed the frontend code for slugs and admin-groups as it was a bit premature to add it in my last PR when the functionality is not implemented. Will be added later.

tl:dr; not perfect - but it works and I'd rather iterate on it over time

## Visual preview  
This is the backend of #857, with only minor changes to the frontend.

Fixes ABA-217, #6, #7

### CODE REVIEW

File: frontend/src/routes/ManageAdmissions/CreateAdmission.tsx [hunk 1]
```diff
@@ -155,13 +179,22 @@ const CreateAdmission: React.FC = () => {
...
         </InputWrapper>
       </FormGroup>
-      <LegoButton type="submit">Opprett opptak</LegoButton>
+      {returnedData && (
+        <FormGroup>
+          <ResultText type={returnedData.type}>
+            {returnedData.message}
+          </ResultText>
+        </FormGroup>
+      )}
```
Comment (LudvigHz) [inline frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:194]: I don't _really_ like this error handling stuff with the state. This really feels like it should be some global snackbar or something similar and not handled here.

 But if you don't want to change it now that's fine.

**Review by LudvigHz — APPROVED** [review]
Looks really good :100:
Not much to complain about :) :shipit:

File: frontend/src/routes/ManageAdmissions/CreateAdmission.tsx (hunk 1 above)
Comment (norbye) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:194]: True, that would be a lot nicer to have, both here and when deleting an application. Adding a new issue to take a look at that.

File: frontend/src/routes/ManageAdmissions/CreateAdmission.tsx [hunk 2]
```diff
@@ -1,40 +1,98 @@
...
+      open_from: formatCurrentDate(),
+      public_deadline: formatCurrentDate(),
+      application_deadline: formatCurrentDate(),
       groups: [],
     },
     onSubmit: (values) => {
-      console.log(JSON.stringify(values, null, 2));
+      setReturnedData(undefined);
+      const processedValues = { ...values };
+      // Convert from iso to iso to re-add timezone data that input[type=datetime-local] removes
```
[the remark this answers is not in the document]
Comment (norbye) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:58]: Tried to find an alternative - but I had some trouble finding something good.. As far as I can tell there aren't really any good alternatives in native HTML5, so if not this then we'd have to add another dependency to sort it out

[the remark this answers is not in the document]
Comment (LudvigHz) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:58]: Hmm yeah okay. Guess that's fine. But then it should be clear what timezone they are inputting here right? Or will luxon add the timezone stuff? (I guess also the comment isn't super clear :upside_down_face: )

[the remark this answers is not in the document]
Comment (norbye) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx:58]: Fair, I'll think of something to improve the comment. The way it works now is that Luxon takes the local timezone as default when running fromISO() and a timezone is not passed (as datetime-local does), and returns the ISO string including the timezon

[the remark this answers is not in the document]
Comment (norbye) [reply frontend/src/routes/ManageAdmissions/CreateAdmission.tsx]: Updated it now to explicitly type out the default option variables that matter to make it a bit more readable what actually happens
